src/BaGet.UI/src/DisplayPackage.tsx

```diff
@@ -177,7 +193,24 @@ class DisplayPackage extends React.Component<IDisplayPackageProps, IDisplayPacka
             </div>
 
             {/* TODO: Fix this */}
-            <div dangerouslySetInnerHTML={ {__html: this.state.package.readme} } />
+            <div dangerouslySetInnerHTML={{ __html: this.state.package.readme }} />
+
+            <div>
+                <h3>Dependencies</h3>
+
+                {this.state.package.dependencyGroups.map(depGroup => (
+                    <div>
+                        <h4>{depGroup.targetFramework}</h4>
+                        <ul>
+                            {depGroup.dependencies.map(dep => (
```

Comment:
![image](https://user-images.githubusercontent.com/1027111/49699690-a1251280-fbd4-11e8-8c34-fe0e3ad9557a.png)

It doesn't throw. Perhaps because there are no `dependencyGroups` ?

Reply:
OTOH: a "no data" case is needed.

Reply:
I've found that `dependencyGroups` is `[]` when no data is present. I now have this, but I'm split between simply not showing anything, and showing a "no dependencies listed" message.

Reply:
nuget.org shows `This package has no dependencies`. Let's stick with that. See: https://www.nuget.org/packages/Newtonsoft.Json/3.5.8

Reply:
Yes. I've added and pushed, but not tested. I think this should work though (given react's docs). Could you try it out, @loic-sharma ?

Currently at work; limited time.

Reply:
![image](https://user-images.githubusercontent.com/1027111/49833750-bb105200-fd9a-11e8-81de-8ca83950d5a2.png)

![image](https://user-images.githubusercontent.com/1027111/49833761-c3688d00-fd9a-11e8-9108-61be2cf39704.png)

Should be good now.